api/api.yaml

```diff
@@ -179,4 +179,4 @@ definitions:
         type: string
         description: >
           If this search result represents a facet, facet_value is the empty string.
-          If this search result represents a facet and selected value, facet_value is the selected value.
\ No newline at end of file
+          If this search result represents a facet and selected value, facet_value is the selected value.
```

Comment:
Changes to end-to-end tests:
- Change "Field search" test name to "Search box - click on drop-down row"
- Add "Search box - chips" test.
  - Click on female facet value
  - Assert chip
  - Unselect female facet value
  - Assert no chip
  - Click on female facet value again
  - Click on x in chip
  - Assert female facet unchecked

Reply:
DOne